snagsby/sources.py

```diff
@@ -4,15 +4,22 @@
 import logging
 import re
 
+import boto3
+import botocore
+from botocore.client import Config
+
+from .registry import Registry
+
 try:
     from urlparse import urlparse, parse_qs
 except ImportError:
     from urllib.parse import urlparse, parse_qs
 
+try:
+    from json.decoder import JSONDecodeError
+except ImportError:
+    JSONDecodeError = ValueError
```

Comment:
It would be nice to have a comment here explaining what this is a compatibility shim for, so that it's easier to know when we can safely remove it in the future.